dist/mixins/utility/utility-mixins.less

```diff
@@ -31,3 +31,15 @@
 .customColorProperty(@token) {
     .customProperty(color, @token);
 }
+
+.customBackgroundColorProperty(@token) {
```

Comment:
It would be cool (but not necessary in this PR) for each of these to handle the prefix/suffix of the color variables. That way we can have consistent names
My idea:
In badge.less:
```
.customBackgroundColorPropery(badge)`
```

In the mixin
```
.customBackgroundColorPropery(@token) {
   .customProperty(background-color, ~"@{token}-background-color")
}
```

Saves some typing and makes it so all background color variables have to end in "background-color"

Reply:
Nice idea. Our variables fall into 3 categories: Global, Product and Component. A solid cross-platform naming convention is beginning to take shape, and I think we should use your idea to help inform and shape that convention.